packages/graphiql-toolkit/src/create-fetcher/createFetcher.ts

```diff
@@ -40,7 +41,13 @@ export function createGraphiQLFetcher(options: CreateFetcherOptions): Fetcher {
           graphQLParams.operationName || undefined,
         )
       : false;
+
     if (isSubscription) {
+      if (options.sseUrl) {
```

Comment:
or maybe instead of new argument `sseUrl` check by `ws` protocol like in `graphql-helix`?
```suggestion
      if (!options.subscriptionUrl.startsWith('ws')) {
```
https://github.com/contra/graphql-helix/blob/f8e9b1acde278d3526bb43a6d573383d037e4993/packages/graphiql/src/index.tsx#L119